visual-studio-vstsbuildagent-vm/azuredeploy.parameters.json

```diff
@@ -18,22 +18,19 @@
       "value": "GEN-UNIQUE"
     },
     "vmSize": {
-      "value": "Standard_DS2"
+      "value": "Standard_DS1"
```

Comment:
put defaultValues in the template and remove other values not needed by CI (so the use is prompted for relevant values.